### DISCUSSION TITLE
Liquid level sensors

### DISCUSSION THREAD

**Discussion by mgrouch** [bareboat-necessities/lysmarine_gen#370] [Ideas]
Non-Contact Liquid Level Sensor Water Level Switch XKC-Y25-NPN for Arduino EK1954
For plastic tanks

https://www.amazon.com/dp/B086QX726M/

**Comment by PtJudeRI** [comment]
Do you need several on a tank to indicate various levels?  Would be a much cheaper alternative than some of the Maretron solutions.

**Reply by mgrouch** [reply]
Yes you would need several ones
